frontend/pluginloader.lua

```diff
@@ -85,14 +93,64 @@ function PluginLoader:loadPlugins()
     end
 
     -- set package path for all loaded plugins
-    for _,plugin in ipairs(self.plugins) do
+    for _,plugin in ipairs(self.enabled_plugins) do
         package.path = string.format("%s;%s/?.lua", package.path, plugin.path)
         package.cpath = string.format("%s;%s/lib/?.so", package.cpath, plugin.path)
     end
 
-    table.sort(self.plugins, function(v1,v2) return v1.path < v2.path end)
+    table.sort(self.enabled_plugins, function(v1,v2) return v1.path < v2.path end)
+
+    return self.enabled_plugins, self.disabled_plugins
+end
+
+function PluginLoader:genPluginManagerSubItem()
+    local enabled_plugins, disabled_plugins = {}, {}
+    if self.all_plugins == nil then
+        self.all_plugins = {}
+        enabled_plugins, disabled_plugins = self:loadPlugins()
+    end
+
+    for _, plugin in ipairs(enabled_plugins) do
+        local element = {}
+        element.fullname = plugin.fullname or plugin.name
+        element.name = plugin.name
+        element.enable = true
+        table.insert(self.all_plugins, element)
+    end
+
+    for _, plugin in ipairs(disabled_plugins) do
+        local element = {}
+        element.fullname = plugin.fullname or plugin.name
+        element.name = plugin.name
+        element.enable = false
+        if element.name ~= "storagestat" then
```

Comment:
May be we could put "storagestat" in a module local
```
local PLUGINS_OBSOLETE = {
    "storagestat"=true,
}
```
so it's no more harcoded there and above (where you added it 1 or 2 weeks ago).

Reply:
Another module or where to put PLUGINS_OBSOLETE?

Reply:
In this module, just under `local DEFAULT_PLUGIN_PATH = "plugins"`.
```lua
local DEFAULT_PLUGIN_PATH = "plugins"
local OBSOLETE_PLUGINS = {
    "storagestat" = true,
}
```

Reply:
Thanks!